### PR TITLE
Pet-Tracker Front End Updates

### LINKED ISSUE #18: Pets Index - Front End
## Description
Now that we have boostrap live, let's rework the front end for the pets index (View All Pets) resource.

### LINKED ISSUE #21: New Pet - Front End
## Description
Let's rework the front end for the new pet resource.

### LINKED ISSUE #56: Header - Front End
## Description
Let's rework the Header and navigation menu's Front End so that the app name is in the top-left, and all nav options are top-right.

### PR DESCRIPTION
**Pull request by tony-opp**
Fix: #21 
Fix: #56 
Fix: #18 

## PR Checklist
Front End Updates
- I updated the header/navigation menu/footer
- I updated the 'New Pet' and 'All Pets' pages so that the main content container is consistent
- Updated 'All Pets' page to show each pet as a card/tile (the button for each pet still needs to be re-linked)
- Made sure everything is mobile responsive
fix: 
- [ ] Updated documentation (if necessary)

### CODE REVIEW

**Comment by tony-opp** [comment]
The Pet-Front-End branch was successfully merged with the master before creating this Pull Request.

File: app/views/pet/index.html.erb [hunk 1]
```diff
@@ -1,14 +1,30 @@
-<div class="content">
```
Comment (mike-webster) [inline app/views/pet/index.html.erb:1]: Can you switch this back to 2 spaces for tabs as opposed to the 500 you seem to be using here? 😆

File: app/views/pet/index.html.erb [hunk 2]
```diff
@@ -1,14 +1,30 @@
...
-        </div>
+			<div class="col-sm-4 mb-3">
+				<div class="card shadow-sm">
+					<div class="card-body">
+						<h5 class="card-title"><%= p.name %></h5>
+						<p class="card-subtitle text-muted mb-4">
+            <%= p.breed %><br>
+            </p>
+            <div class="btn-group btn-group-sm" role="group" aria-label="Basic example">
+              <a href="<%= new_pet_path %>" class="btn btn-secondary">more info</a>
```
Comment (mike-webster) [inline app/views/pet/index.html.erb]: Clicking on "more info" brings the user to create a new path? 🤔

File: app/views/pet/new.html.erb [hunk 3]
```diff
@@ -1,78 +1,82 @@
...
-  <%= form_with(model: @pet, local: true, url: "/pet", method: "post") do |form| %>
-    <div class="row">
-      <% if @pet.errors[:name].any? %>
-        <div class="col-4 text-center error_explanations">
-          <p> <%= @pet.errors[:name].first %> </p>
-        </div>
-      <% else %>
-        <div class="col-4"></div>
-      <% end %>
+<div class="container mt-4">
```
Comment (mike-webster) [inline app/views/pet/new.html.erb:1]: Can you set the spacing back to 2 spaces here too?

**Review by mike-webster — CHANGES_REQUESTED** [review]
Just a couple minor things - everything else looks good!

**Comment by mike-webster** [comment]
K. I'm just gonna do it for you 😄

**Review by mike-webster — APPROVED** [review]

**Pull request merged**